app/src/main/res/values/colors.xml

```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="purple_200">#FFBB86FC</color>
-    <color name="purple_500">#FF6200EE</color>
-    <color name="purple_700">#FF3700B3</color>
-    <color name="teal_200">#FF03DAC5</color>
-    <color name="teal_700">#FF018786</color>
+    <color name="ffbb86fc">#FFBB86FC</color>
+    <color name="ff6200ee">#FF6200EE</color>
+    <color name="dd3700b3">#FF3700B3</color>
+    <color name="ff03dac5">#FF03DAC5</color>
+    <color name="ff018786">#FF018786</color>
+    <color name="e2102e">#E2102E</color>
```

Comment:
tức là đặt tên theo mã màu ấy. 
http://chir.ag/projects/name-that-color/#6195ED